dind/templates/statefulset.yaml

```diff
@@ -56,6 +56,10 @@ spec:
                 fieldRef:
                   apiVersion: v1
                   fieldPath: metadata.name
+            {{- range $key, $value := .Values.extraEnvVariables }}
```

Comment:
Not a fan of the variable name, prefer `extraEnvVars` (the standard followed by `bitnami` charts). This should be wrapped in e.g. `{{- if .Values.extraEnvVars }} ... {{- end }}` otherwise it makes passing extra environment variables a requirement, e.g. a slightly modified snippet from bitnami:
```
          env:
            # ...
            {{- if .Values.extraEnvVars }}
            {{- toYaml .Values.extraEnvVars | nindent 12 }}
            {{- end }}
          {{- if or .Values.extraEnvVarsCM .Values.extraEnvVarsSecret }}
          envFrom:
            {{- if .Values.extraEnvVarsCM }}
            - configMapRef:
                name: {{ .Values.extraEnvVarsCM }}
            {{- end }}
            {{- if .Values.extraEnvVarsSecret }}
            - secretRef:
                name: {{ .Values.extraEnvVarsSecret }}
            {{- end }}
          {{- end }}
```

Reply:
Seems useful

Reply:
Added the if @masseybradley